app/frontend/GetModel.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+protoc --proto_path=3rdparty/onnx/onnx --cpp_out=$1 onnx.proto
+yolo export model=yolo11x.pt format=onnx save_dir=$1
```

Comment:
ultralytics was complaining that `save_dir` flag does not exist (at least on my machine). If it is not a problem for you, please, tell me what is your `ultralytics` version
This replacement worked for me, you can try as well:
```suggestion
yolo export model=$1/yolo11x.pt format=onnx
```